minpy/nn/solver.py

```diff
@@ -4,35 +4,39 @@
 from __future__ import print_function
 import time
 
+import minpy
+
 from minpy.nn import optim, init
 from minpy import core
 import minpy.numpy as np
 
+from minpy.visualize.writer import LegacySummaryWriter as SummaryWriter
+import minpy.visualize.summaryOps as summaryOps
 
 class Solver(object):
     """
     A Solver encapsulates all the logic necessary for training classification
     models. The Solver performs stochastic gradient descent using different
     update rules defined in optim.py.
-
+    
```

Comment:
@jermainewang Also I haven't merged the master branch and visualize branch of my forked repo while I am trying to merge the visualize branch of my forked repo with the visualize branch of the minpy repo. If you try the files in the master branch of my forked repo, you will meet many problems that have been fixed in the visualize branch.